src/components/engagement/engagement.service.ts

```diff
@@ -106,45 +106,6 @@ export class EngagementService {
     @Logger(`engagement.service`) private readonly logger: ILogger
   ) {}
 
-  // HELPER //////////////////////////////////////////////////////////
-
-  propMatch = (query: Query, property: string, baseNode: string) => {
-    const readPerm = 'canRead' + upperFirst(property);
-    const editPerm = 'canEdit' + upperFirst(property);
-    query.optionalMatch([
-      [
-        node('requestingUser'),
-        relation('in', '', 'member'),
-        node('sg', 'SecurityGroup'),
-        relation('out', '', 'permission'),
-        node(editPerm, 'Permission', {
-          property,
-          edit: true,
-        }),
-        relation('out', '', 'baseNode'),
-        node(baseNode),
-        relation('out', '', property, { active: true }),
-        node(property, 'Property'),
-      ],
-    ]);
-    query.optionalMatch([
-      [
-        node('requestingUser'),
-        relation('in', '', 'member'),
-        node('sg', 'SecurityGroup'),
-        relation('out', '', 'permission'),
-        node(readPerm, 'Permission', {
-          property,
-          read: true,
-        }),
-        relation('out', '', 'baseNode'),
-        node(baseNode),
-        relation('out', '', property, { active: true }),
-        node(property, 'Property'),
-      ],
-    ]);
-  };
-
   // helper method for defining properties
   property = (prop: string, value: any | null, baseNode: string) => {
```

Comment:
This needs to be replaced

Reply:
I think this helper function will not be used anymore since we refactor all the services with new abstractions methods. Only several services contain this helper function and they are doing property label changing in this method... so I think it's a little difficult to replace now.

Reply:
I'm confused. Are you saying you want to wait until these create() queries are refactored? Do we have a newer abstraction for them?